src/DynamicSearchBundle/Resources/config/services.yml

```diff
@@ -8,6 +8,8 @@ services:
         autoconfigure: true
         public: false
 
+    DynamicSearchBundle\Service\LockServiceInterface: '@DynamicSearchBundle\Service\LockService'
```

Comment:
it complained about autowiring on services that don't exist

Reply:
BTW: Public Bundles should not rely on autowiring: https://symfony.com/doc/current/service_container/autowiring.html#public-and-reusable-bundles